randovania/games/dread/logic_database/Burenia.txt

```diff
@@ -258,7 +258,7 @@ Extra - asset_id: collision_camera_002
           Speed Booster
           # A grounded bomb jump with cross bombs to get past the fan
           Movement (Beginner) and Lay Cross Bomb
-          Flash Shift and Movement (Advanced)
+          Flash Shift and Flash Shift Upgrade and Movement (Advanced)
```

Comment:
I think we arrived on the conclusion that this can be lowered from advanced to intermediate. Since you're changing this line right here, it probably makes more sense to change that in this PR to avoid merge conflicts.